syntax = "proto3";

package aivoice.common;

// Common message types used across services

message DialogRef {
  string dialog_id = 1;
  int32 turn = 2;
}

message Timestamp {
  string iso8601 = 1;
}

message ServiceStatus {
  string service_name = 1;
  bool healthy = 2;
  string status_message = 3;
  Timestamp last_check = 4;
}